src/utils/index.tsx

```diff
@@ -191,3 +183,23 @@ export const createClickHandler = (
         }
     };
 };
+
```

Comment:
wanted to try out a helper to make component initialization in tests a little nicer/discourage prop-spreading; what do you think?

Reply:
needs a better name, that's for sure